classes/local/renderer.php

```diff
@@ -127,6 +126,28 @@ public function get_html(): string {
         return $content;
     }
 
+    /**
+     * Renders the template label with the data and returns the result.
+     *
+     * @return string
+     */
+    public function get_name(): string {
```

Comment:
Just leaving this here for completeness, this seems to have caused a regression in the roles list datasource
```
Exception - Call to undefined function mod_cms\local\datasource\user_get_users_by_id()

More information about this error
error
Error
Debug info:
Error code: generalexceptionmessage
error
Error
Stack trace:

    line 98 of /mod/cms/classes/local/datasource/roles.php: Error thrown
    line 59 of /mod/cms/classes/local/renderer.php: call to mod_cms\local\datasource\roles->get_data()
    line 141 of /mod/cms/classes/local/renderer.php: call to mod_cms\local\renderer->get_data()
    line 105 of /mod/cms/classes/local/lib.php: call to mod_cms\local\renderer->get_name()
    line 109 of /mod/cms/lib.php: call to mod_cms\local\lib::add_instance()
    line 137 of /course/modlib.php: call to cms_add_instance()
    line 170 of /course/modedit.php: call to add_moduleinfo()
```

Reply:
Apparently, usr/lib.php is a file that is loaded as-needed, rather than universally, so a 'require_once' is needed.